spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/procedures/ExpireSnapshotsProcedure.java

```diff
@@ -93,6 +97,12 @@ public InternalRow[] call(InternalRow args) {
     Long olderThanMillis = args.isNullAt(1) ? null : DateTimeUtil.microsToMillis(args.getLong(1));
     Integer retainLastNum = args.isNullAt(2) ? null : args.getInt(2);
     Integer maxConcurrentDeletes = args.isNullAt(3) ? null : args.getInt(3);
+    if (maxConcurrentDeletes != null) {
+      LOG.warn(
```

Comment:
I don't think this description is accurate. I believe this property will have no impact now.

Reply:
It will still work but only if the IO does not support bulk

Reply:
Moving this message to the application of the parameter and having it not applied if the underlying IO supports bulks. Message contents changing as well.